### PR TITLE
ReInvent ML Recap

### PR DESCRIPTION
**Pull request by RonakReyhani**

### CODE REVIEW

File: _posts/2023-01-27-re-invent-recap-ml.markdown [hunk 1]
```diff
@@ -0,0 +1,165 @@
...
+
+# AWS ML infrastructure and Services
+![AWS Machine Learning Stack](/img/reinventRecap2022-ML/aws-ml-stack.png)
+
+## ML Compute
+As the graphic illustrates, the bottom layer contains the compute, networking and storage services that construct the foundational block of ML infrastructure on AWS. 
+Here are some recently added capabilities to Compute component in Machine Learning solutions on AWS.
+
+### Amazon EC2 inf2 Instances (Preview)
+As per AWS documentations, Inf2 instances are powered by up to 12 AWS Inferentia2, the third AWS-designed DL accelerator. Inf2 instances offer 3x higher compute performance, up to 4x higher throughput, and up to 10x lower latency compared to Inf1 instances. The Inf2 instances are designed to deliver higher performance at the lowest cost for Deep Learning inference applications. You can now efficiently deploy a 175B parameter model for inference across multiple accelerators on a single Inf2 instance.
```
Comment (carter293) [inline _posts/2023-01-27-re-invent-recap-ml.markdown:34]: I love the blog Nadia ! 
175B is massive number of parameters - did they talk about the cost of training at all?

File: _posts/2023-01-27-re-invent-recap-ml.markdown [hunk 2]
```diff
@@ -0,0 +1,165 @@
...
+// TO DO : Add ML Diagram for end to end workflow
+// TO DO: Make this section more academic and descriptive according to AWS documents
+
+As you know, The end to end workflow of a Machine Learning solution usually consist of the following task:
+
+- Data Acquisition: Explore Data source, extract, pre-process the raw data.
+- Data Transformation: Enrich data and convert data format compatible with the input type of your ML training Algorithm.
+- Visualization: Visualize the training data for comparison, relationship, distribution and composition analysis on your data.  
+- Model Training: Train your model with transformed dataset. You can always set Testing metrics to monitor your model during training.
+- Evaluate your model with test dataset. AWS SageMaker provides you with evaluating metrics.
```
Comment (temyers) [inline _posts/2023-01-27-re-invent-recap-ml.markdown:73]:
```suggestion
- Evaluation: Evaluate your model with test dataset. AWS SageMaker provides you with evaluating metrics. 
```

File: _posts/2023-01-27-re-invent-recap-ml.markdown [hunk 3]
```diff
@@ -0,0 +1,165 @@
...
+This year for the very first time, I attended the AWS Re:invent conference through AWS [All Builders Welcome grant](https://reinvent.awsevents.com/community/all-builders-welcome/). As a machine learning enthusiasm I could participate a few chalktalks in ML/AI learning track as well as Re:invent Announcement sessions. 
+
+In this blog post, I would like to share highlights of some of the cool stuff that are now available or still in preview for Machine learning and data scientist within AWS AI/ML services and more specifically Amazon SageMaker.
+
+To start with, let's look at the end to end workflow of building a Machine Learning solution on AWS and then walk through the new announcements for some of the components in AWS Machine learning architecture. 
+
+
+
+# AWS ML infrastructure and Services
+![AWS Machine Learning Stack](/img/reinventRecap2022-ML/aws-ml-stack.png)
```
Comment (temyers) [inline _posts/2023-01-27-re-invent-recap-ml.markdown:27]: What is the license on using this image?

File: _posts/2023-01-27-re-invent-recap-ml.markdown (hunk 1 above)
Comment (RonakReyhani) [reply _posts/2023-01-27-re-invent-recap-ml.markdown:34]: No in specific I guess hence it's still in preview, user can request for usage and price inquiries, however it's been said that it delivers a better price performance than Inf1 for smaller models!

File: _posts/2023-01-27-re-invent-recap-ml.markdown [hunk 4]
```diff
@@ -0,0 +1,165 @@
...
+image: /img/reinventRecap2022-ML/reinventBanner.jpg
+---
+
+# What's AWS Re:invent?
+
+[AWS Re:Invent](https://reinvent.awsevents.com/) is the biggest tech conference and cloud event of the year. It's billed as "The most transformative event in tech", a learning conference hosted by AWS where the global cloud community come together to meet, learn and get inspired! 
+AWS Re:invent event has a lot to offer in terms of different tracks and area of interests to follow related to your job domain. Such as Machine learning/AI, storage, serverless, etc,...
+As well as variety of session types i.e workshops to chalktalks with different levels from level-100 (Foundational) to level-400 as expert level! 
+
+Well, if you have not heard about AWS Re:invent or if you couldn't attend it for any reason, that means you have probably missed the AWS Re:invent 2022. That's ok! Good thing I like about Tech world is that there are always people who have attended the events, meet-ups and conferences and most of them are always willing to share their experience including their tech take away from the event.
```
[the remark this answers is not in the document]
Comment (RonakReyhani) [reply _posts/2023-01-27-re-invent-recap-ml.markdown:16]: sure, appreciate it Tim, will rethink about this bit and apply the techniques have been mentioned in the google doc to improve this bit.

File: _posts/2023-01-27-re-invent-recap-ml.markdown [hunk 5]
```diff
@@ -0,0 +1,165 @@
...
+As well as variety of session types i.e workshops to chalktalks with different levels from level-100 (Foundational) to level-400 as expert level! 
+
+Well, if you have not heard about AWS Re:invent or if you couldn't attend it for any reason, that means you have probably missed the AWS Re:invent 2022. That's ok! Good thing I like about Tech world is that there are always people who have attended the events, meet-ups and conferences and most of them are always willing to share their experience including their tech take away from the event. 
+
+This year for the very first time, I attended the AWS Re:invent conference through AWS [All Builders Welcome grant](https://reinvent.awsevents.com/community/all-builders-welcome/). As a machine learning enthusiasm I could participate a few chalktalks in ML/AI learning track as well as Re:invent Announcement sessions. 
+
+In this blog post, I would like to share highlights of some of the cool stuff that are now available or still in preview for Machine learning and data scientist within AWS AI/ML services and more specifically Amazon SageMaker.
```
[the remark this answers is not in the document]
Comment (RonakReyhani) [reply _posts/2023-01-27-re-invent-recap-ml.markdown:20]: sure will rework on this bit

File: _posts/2023-01-27-re-invent-recap-ml.markdown (hunk 3 above)
Comment (RonakReyhani) [reply _posts/2023-01-27-re-invent-recap-ml.markdown:27]: there is nothing specific saying you can not use it, however I still added the link to the article in resource section. would that be enough?

File: _posts/2023-01-27-re-invent-recap-ml.markdown [hunk 6]
```diff
@@ -0,0 +1,165 @@
...
+
+## Recap of Re:Invent Recap
+
+The innovation in AWS Cloud Services never ends, every year at Re:Invent we get surprised by new announcements. In this article, I mainly focused on compute, data preparation, build, train and deploy on the most popular ML service Amazon SageMaker at a general level.
+This post was not technical showing how to use the new capabilities, neither covered all the announcements for Machine learning. So if you are interested to know more about what's been recently announced please make sure to check the [AWS announcement platform](https://aws.amazon.com/about-aws/whats-new/2022/11/) to read more and in depth about what are the new capabilities or feature for AWS ML services. 
+
+More Amazon SageMaker related blog post coming up, Stay tuned with our posts if you are keen to learn how to "Tune your model with hyperparameter range" in Amazon SageMaker cost effectively or want to discover "what's under the hood of AWS DeepRacer" with Amazon SageMaker.
```
Comment (zmaleki) [inline _posts/2023-01-27-re-invent-recap-ml.markdown:160]:
```suggestion
More Amazon SageMaker related blog post coming up. Stay tuned with our future posts If you are keen to learn how to "Tune your model with hyperparameter range" in Amazon SageMaker cost effectively or want to discover "what's under the hood of AWS DeepRacer" with Amazon SageMaker.
```

**Comment by Hamo69** [comment]
@RonakReyhani is this going ahead or shall we remove?

**Comment by RonakReyhani** [comment]
> @RonakReyhani is this going ahead or shall we remove?

Hey- I have applied the feedback and change requests, can go ahead if we are willing to publish?

**Comment by carter293** [comment]
I like it :D

**Review by carter293 — APPROVED** [review]

**Pull request merged**